Makefile

```diff
@@ -217,6 +217,7 @@ dev: ## Set Environment to DEV
 	$(eval export ENABLE_MORNING_DEPLOYMENT=true)
 	$(eval export SLIM_DEV_DEPLOYMENT ?= true)
 	$(eval export CA_ROTATION_EXPIRY_DAYS ?= 360)
+	$(eval export ENABLE_PAAS_ADMIN_CONTINUOUS_DEPLOY ?= true)
```

Comment:
does this work!? I thought you couldn't `export` envs to the calling shell from within a makefile, because it's running its own subshel?

Reply:
Doesn't export it to the calling shell, I don't think. It exports it for all of the scripts run by Make.

Reply:
aaah, all subshells. That totally makes sense now.